src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/InputIllegalTypeArrays.java

```diff
@@ -0,0 +1,19 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.illegaltype;
+
+public class InputIllegalTypeArrays {
```

Comment:
please follow comments pattern of InputIllegalTypeSameFileName.java, to define configuration, and mark by comment each line with violation.

Reply:
done